core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3759,6 +3760,21 @@ class KafkaApis(val requestChannel: RequestChannel,
     CompletableFuture.completedFuture[Unit](())
   }
 
+  // Just a placeholder for now.
+  def handleListClientMetricsResources(request: RequestChannel.Request): CompletableFuture[Unit] = {
```

Comment:
Why does this return `CompletableFuture[Unit]` instead of `Unit`?

Reply:
Just giving the option of asynchronous completion. However, since I don't strictly need it now, I am simplifying it to `Unit`.